medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/model/Appointment.java

```diff
@@ -19,4 +22,16 @@ public class Appointment {
     private Patient patient;
     private LocalDateTime startDate;
     private LocalDateTime endDate;
+    @Override
+    public boolean equals(Object o) {
```

Comment:
wszystkie metody equals ktore maja zwykle id mozesz napisac troszke krocej/lepiej 


    public boolean equals(Object o) {
        if (this == o) return true;

        if (!(o instanceof Appointment other)) return false;

        return id != null && id.equals(other.getId());
    }